src/crimson/os/cyanstore/cyan_store.cc

```diff
@@ -276,6 +276,20 @@ CyanStore::omap_get_values(
     true, values);
 }
 
+seastar::future<ceph::bufferlist>
+CyanStore::omap_get_header(
+    CollectionRef ch,
+    const ghobject_t& oid
+  ) {
+  auto c = static_cast<Collection*>(ch.get());
+  auto o = c->get_object(oid);
+  if (!o)
```

Comment:
personally, i'd suggest add curly braces even for single statement. it's more error-prone.